qiskit/circuit/instruction.py

```diff
@@ -162,6 +163,9 @@ def params(self, parameters):
             # example: snapshot('label')
             elif isinstance(single_param, str):
                 self._params.append(single_param)
+            # example: Aer expectation_value_snapshot
```

Comment:
Not a blocker (more of a question), but thinking about this comment a bit, maybe putting a bit more detail in how this gets used, maybe something like would be helpful:

```suggestion
            # example: Aer expectation_value_snapshot [complex, 'X']
```